compiler/fir/analysis-tests/testData/resolve/intersectionTypes.kt

```diff
@@ -5,7 +5,7 @@ interface B
 class Clazz1 : A, B
 class Clazz2 : A, B
 
-fun <K> select(x: K, y: K): K
+<!NON_MEMBER_FUNCTION_NO_BODY!>fun <K> select(x: K, y: K): K<!>
```

Comment:
This and below are different. Actually, not trivial to _not_ introduce other errors while removing `NON_MEMBER_FUNCTION_NO_BODY` case.